ptest/radio_session.py

```diff
@@ -89,6 +90,9 @@ def __init__(self, device_name, imei, uplink_console, port, send_queue_duration,
         except Exception as e:
             print("Not connected to email")
 
+        if os.path.exists("uplink.json"):
+            os.remove("uplink.json")
```

Comment:
perhaps add for http_uplink as well
```suggestion
            os.remove("uplink.json")
        if os.path.exists("http_uplink.json"):
            os.remove("http_uplink.json")
```